### DISCUSSION TITLE
Google Sheets Data Source Availability

### DISCUSSION THREAD

**Discussion by danderson2692** [Budibase/budibase#6873] [Help]
I've seen the documentation that there is a data source configured which allows you to connect to a google sheet. I'm wondering if this has been disabled, or if it's only available if you self host? I've configured google authentication but I don't see it in my list of available external data sources in my cloud-hosted app:
![image](https://user-images.githubusercontent.com/16950976/180657553-d7121948-d383-4c81-949e-887f181cf673.png)

**Comment by melohagan** [comment]
Hi @danderson2692 

I can see the Google Sheets connector on both cloud and self-hosted. Is your cloud-hosted app up-to-date with the latest version?

Could you provide your tenant id please? E.g. `<tenant-id>.budibase.com`

**Reply by danderson2692** [reply]
Hi @melohagan,

I've tried updating my app to the newest version (running 1.1.21) and creating a new app but I still see the same behavior in the new app too. I also have an app that I created from a template which is running 1.1.7 which also doesn't show it. Is it possible that I configured my google authentication incorrectly, or is there something else that could be preventing it from showing?

My tenant is dog.budibase.com

Thanks for the help!

**Reply by danderson2692** [reply]
On the off chance that this documentation was right rather than what's shown in the Google Auth screen, I tried updating a callback my google credentials to end with /api/admin/auth/google/callback but it still appears the same.
https://docs.budibase.com/docs/sso-with-google

**Reply by shogunpurple** [reply]
Hi @danderson2692 - the google sheets integration will not work in the cloud as we are still waiting for google to verify us. This is actually a mistake (that the google sheets integration is being shown, it should only show in self host) we hope to have google sheets support in cloud very soon.

**Reply by danderson2692** [reply]
@shogunpurple gotcha, thank you! Is there an issue or somewhere I can keep an eye on to look for progress?

**Reply by shogunpurple** [reply]
https://github.com/Budibase/budibase/issues/4234
